README.md

```diff
@@ -389,8 +391,6 @@ $ mvn package # Packages the application examples into a standalone jar
 | [4.0.0-post](../../../tree/4.4.0-post/) | 4.0.0              | 1.0.0             |
 | [3.3.0-post](../../../tree/3.3.0-post/) | 3.3.0              | 0.11.0            |
 
-\*You must manually build the `2.3` version of Apache Kafka and the `5.3.x` version of Confluent Platform.  See instructions above.
-
```

Comment:
Why remove this? Should have been updated to `2.4` I guess (and should be updated to `2.5` now?)

Reply:
I removed the footnote because with this change there is no SNAPSHOT version anymore in the matrix, i.e. no footnote is needed anymore. It could be confusing having a footnote w/o a reference to it.

What we should do in any case is making sure that the footnote is correct on master and on the 5.5.x branch that will be cut from it before the next release, because on those branches we will have again a SNAPSHOT version.

Reply:
Well, we would also have a `5.4.1-SNAPSHOT` on `5.4.x` branch -- but I guess you imply that we should not refer to snapshot versions for those, what sounds reasonable to me.

For this case, we would only have a snapshot reference on `master` because `5.5.x` will only be cut after code freeze.